source/mainnet/net/resources/release-notes-mainnet.rst

```diff
@@ -12,6 +12,14 @@ Release notes
 Mainnet 3: Alpha Centauri 3.0
```

Comment:
Some of the other releases just have their own "Mainnet X: Alpha Centauri X.Y" titles, so in that format this would be "Mainnet 3: Alpha Centauri 3.1" and it wouldn't listed under the same title as recent client release. But I like the way this release note has been entered better tbh, under the same title as the client release, but with a different date.

I'm not sure which one to go for.